### PR TITLE
chore: update Electron reps

### PR DESCRIPTION
**Pull request by bnb**
updates one Electron rep to @erickzhao, removing @codebytere (thank you @codebytere <3), and updating my personal employment.

### CODE REVIEW

**Review by dylans — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
 
 - **Appium**: Isaac Murchie ([@imurchie](https://github.com/imurchie), Sauce Labs) & Jonah Stiennon([@Jonahss](https://github.com/Jonahss), Origin Labs)
 - **Dojo**: Dylan Schiemann ([@dylans](https://github.com/dylans), Living Spec)
-- **Electron**: Shelley Vohr ([@codebytere](https://github.com/codebytere), Microsoft) & Tierney Cyren ([@bnb](https://github.com/bnb), Microsoft)
+- **Electron**: Erick Shao ([@erickzhao](https://github.com/erickzhao), Slack) & Tierney Cyren ([@bnb](https://github.com/bnb), Twilio)
```
Comment (erickzhao) [inline README.md:100]:
```suggestion
- **Electron**: Erick Zhao ([@erickzhao](https://github.com/erickzhao), Slack) & Tierney Cyren ([@bnb](https://github.com/bnb), Twilio)
```

typo 😅

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bnb) [reply README.md:100]: Ah my sincere apologies.

**Review by erickzhao — APPROVED** [review]

**Review by ljharb — APPROVED** [review]

**Review by VoltrexMaster — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Review by tobie — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by joesepi — APPROVED** [review]

**Review by WaleedAshraf — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mhdawson** [comment]
I assume this can land as it's been open for a long time and has lots of approvals.

**Pull request merged**